Makefile

```diff
@@ -321,20 +321,17 @@ dev-db-openshift: prebuild-check deps generate $(FRESH_BIN)
 
 .PHONY: dev-openshift
 dev-openshift: prebuild-check deps generate build bin/docker/fabric8-auth-linux
-	minishift start --cpus 4
+	minishift start --cpus 3
 	./minishift/check_hosts.sh
-	-eval `minishift oc-env` &&  oc login -u developer -p developer && oc new-project auth-openshift
-	AUTH_DEVELOPER_MODE_ENABLED=true \
-	kedge apply -f minishift/kedge/db-auth.yml
-	sleep 5s
-	-eval `minishift docker-env` && docker login -u developer -p $$(oc whoami -t) $$(minishift openshift registry) && docker build -t fabric8/fabric8-auth:dev bin/docker
-	-kedge delete -f minishift/kedge/auth-local.yml
-	kedge apply -f minishift/kedge/auth-local.yml
+	-eval `minishift oc-env` && oc login -u developer -p developer && oc new-project auth-openshift
+	kedge apply -f minishift/kedge/auth-db.yml
+	-eval `minishift docker-env` && docker build -t fabric8/fabric8-auth:dev bin/docker
+	IMAGE_TAG=dev AUTH_DEVELOPER_MODE_ENABLED="'true'" kedge apply -f minishift/kedge/auth-local.yml
+	-oc rollout latest dc/auth
```

Comment:
this command will automatically pick up the latest built image, but this might fail for the first time we do it so can be ignored